cmd/arena/commands/resource.go

```diff
@@ -77,3 +77,71 @@ func gpuInContainerDeprecated(container v1.Container) int64 {
 
 	return val.Value()
 }
+
+// filter out the pods with resources
+func resourcePods(pods []v1.Pod) (podsWithResource []v1.Pod) {
+	for _, pod := range pods {
+		if gpuInPod(pod) > 0 || cpuInPod(pod) > 0 || memoryInPod(pod) > 0 {
+			podsWithResource = append(podsWithResource, pod)
+		}
+	}
+	return podsWithResource
+}
+
+// The way to get CPU Count of Node
+func cpuInNode(node v1.Node) int64 {
+	val, ok := node.Status.Capacity[v1.ResourceCPU]
```

Comment:
this should be `node.Status.Allocatable[v1.ResourceCPU]`

Reply:
OK, Got it, so the gpu resource function `gpuInNode` and `gpuInContainer` should also be updated?

Reply:
no need to update gpu, as gpu resquests equals to limits always.

Reply:
ok, thanks!

Reply:
Emmm, in my opinion the `Allocatable` means node remaining resource, but here we want to get node's total resource, so we should use `Capacity` here instead of `Allocatable`, right?

Reply:
users should care more about the total resources they can use, not include the reserved resources. it's my opinion.  :)

what's your thought? @cheyang

Reply:
My suggestion is that we should keep consistent with `kubectl describe no`. Take a look at 
https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/internalversion/describe.go#L2773

Reply:
Hi @cheyang In `kubectl describe no`, it seems that the `Capacity` and `Allocatable` are all showed in response message.

Reply:
We should use the same algorithm of `describe node` when getting the results of `total` and `allocated`.  I think we can leverage their implementation, and make sure our result is consistent with it.